src/us/kbase/workspace/database/provenance/CreditMetadata.java

```diff
@@ -210,6 +238,9 @@ public boolean equals(Object obj) {
 	 *                unique persistent ID for the resource (i.e. the source data for this
 	 *                workspace object). Should be in the format
 	 *                <database name>:<identifier within database>
+	 * @param repository
+	 *                the repository (as an {@link Organization}) from which the resource can
```

Comment:
Organization will be automatically linked since it's in the parameter list, so this is redundant, but it doesn't hurt either